packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -1227,8 +1179,8 @@ export function makeWallet({
     return ret;
   }
 
-  /** @type {Store<Payment, PaymentRecord>} */
-  const payments = makeScalarMap('payment');
+  /** @type {Store<number, PaymentRecord>} */
+  const idToPaymentRecord = makeScalarMap('payment');
```

Comment:
```suggestion
  const idToPaymentRecord = makeScalarMap('paymentId');
```
Since the type of the key changed